api/placer.py

```diff
@@ -114,6 +124,16 @@ class TargetedPlacer(Placer):
     An exception is thrown in upload.process_upload() if you try. This could be fixed by making a better schema.
     """
 
+    # TODO: Change schemas to enabled targeted uploads of more than one file.
+    # Ref docs from placer.TargetedPlacer for details.
+    def requireFiles(self):
```

Comment:
Maybe I'm missing it but it doesn't look like this is getting used in the `check()` phase for `TargetedPlacer`